ci/prepare.sh

```diff
@@ -14,7 +14,11 @@ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58
 
 if [ "$INSTALLDOCKER" == "True" ] ; then
     if [ ! -f /etc/apt/sources.list.d/docker.list ] ; then
-        sudo su -c 'echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" > /etc/apt/sources.list.d/docker.list'
+        if [ "$UBUNTUVER" == "14.04" ]; then
+            sudo su -c 'echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" > /etc/apt/sources.list.d/docker.list'
+        elif [ "$UBUNTUVER" == "16.04" ]; then
+            sudo su -c 'echo "deb https://apt.dockerproject.org/repo ubuntu-xenial main" > /etc/apt/sources.list.d/docker.list'
+        fi
```

Comment:
@mestery to be consistent, ca we use ``case $UBUNTUVER `` as the code below? Thanks.